src/components/UserPortal/UserPortalNavigationBar/UserPortalNavigationBar.spec.tsx

```
@@ -1035,9 +1035,8 @@ describe('UserPortalNavigationBar', () => {
       });
 
       // Import NotificationToast to spy on it
-      const { NotificationToast } = await import(
-        'components/NotificationToast/NotificationToast'
-      );
+      const { NotificationToast } =
+        await import('components/NotificationToast/NotificationToast');
```

ESLint

[error] Replace `⏎········await·import('components/NotificationToast/NotificationToast'` with `·await·import(⏎········'components/NotificationToast/NotificationToast'⏎······`

(prettier/prettier)